credentials/gs1-8110-coupon/credential.json

```diff
@@ -0,0 +1,34 @@
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://contexts.vcplayground.org/examples/gs1-8110-coupon/v1.json"
+  ],
+  "type": ["VerifiableCredential", "GS18110CouponCredential"],
+  "issuer": {
+    "id": "did:key:z6MkrHKzgsahxBLyNAbLQyB1pcWNYC9GmywiWPgkrvntAZcj",
+    "name": "Big Retail Store",
+    "url": "https://example.com",
+    "image": "https://vcplayground.org/examples/gs1-8110-coupon/logo.png"
+  },
+  "name": "Digital coupon: Buy one, get one free 16 oz coffees.",
+  "issuanceDate": "2023-06-04T00:00:00Z",
+  "expirationDate": "2023-08-01T08:00:00Z",
```

Comment:
```suggestion
  "expirationDate": "2024-08-01T08:00:00Z",
```

Reply:
I just removed it entirely to avoid the issue.